packages/core/src/index.ts

```diff
@@ -254,16 +293,40 @@ class Hopr extends EventEmitter {
       libp2pSendMessage(this.libp2p, dest, protocol, msg, opts)
     const hangup = this.libp2p.hangUp.bind(this.libp2p)
 
-    this.heartbeat = new Heartbeat(this.networkPeers, subscribe, sendMessageAndExpectResponse, hangup)
+    const protocol_heartbeat = `hopr/${this.environment.id}/heartbeat`
```

Comment:
why is this not in a constants file? Or in heartbeat?

Reply:
because it's not a constant value
I don't want heartbeat to know about environment really, that's why I only pass the bare minimum - the protocol string

Reply:
can we just inline them then, doesn't need to be a variable.

Also variables should be camelCase too.